appConfig/appConfig.dev1.yaml

```diff
@@ -1,15 +1,9 @@
-- key: APPINSIGHTS_CONNECTIONSTRING
-  value: "{{servicename}}-APPINSIGHTS-CONNECTIONSTRING"
+- key: APPINSIGHTS_INSTRUMENTATIONKEY
+  value: "{{servicename}}-APPINSIGHTS-INSTRUMENTATIONKEY"
```

Comment:
If you're on FCP, then none of these files do anything.  All app config is the Platform repo in ADO.